C/plugins/north/OMF/omf.cpp

```diff
@@ -429,22 +434,27 @@ bool OMF::sendDataTypes(const Reading& row, OMFHints *hints)
 		}
 		string errorMsg = errorMessageHandler(e.what());
 
-		Logger::getLogger()->warn("Sending JSON dataType message 'Container' "
-					   "not blocking issue: |%s| - %s - %s %s",
-					   (m_changeTypeId ? "Data Type " : "" ),
-					   errorMsg.c_str(),
-					   m_sender.getHostPort().c_str(),
-					   m_path.c_str() );
+		string msg = "An error occured sending the dataType container message for the asset " + assetName
```

Comment:
Typo: occured -> occurred. This typo occurs in a lot of places.

Reply:
The beauty of copy/paste!